src/argbash-init.m4

```diff
@@ -14,137 +14,114 @@ version=_ARGBASH_VERSION
 # ARG_OPTIONAL_REPEATED([wrap], ,[What script(s) to wrap])
 # ARG_OPTIONAL_SINGLE([mode], m, [The slider between feature-rich and simple script.], [default])
 # ARG_TYPE_GROUP_SET([mode], [MODE], [mode], [default,full,minimal])
-# ARG_VERSION([echo "argbash-init v$version"])
+# ARG_VERSION([echo "argbash-init v${version}"])
 # ARG_HELP([Make a template for scripts.])
 
 # ARGBASH_GO[
 
-
 _variables=()
 HAVE_POSITIONAL_ARG=no
 
-
-# This should be in sync with _translit_var in stuff.m4
-_translit_var()
+# This should be in sync with _translate_var in stuff.m4
+_translate_var()
 {
-	printf "\$_arg_%s" "$1" | tr '[:upper:]' '[:lower:]' | tr '-' '_'
+	printf '${_arg_%s}' "$1" | tr '[:upper:]' '[:lower:]' | tr '-' '_'
 }
 
-
```

Comment:
I got this habit from Python - one blank line in a block can make the code of a e.g. function more readable, while two blank lines clearly separate consecutive parts that are different.
Are two blank lines something Shellcheck warns about?

Reply:
Nope, probably came from the beautify extension, I can clean it up if you want.

Reply:
yes, please, preserve two lines break.

Reply:
I believe I've gone through and consistently applied your style.  Since the original removal was done by a code formatted I turned that off and tried to apply your rules as described consistently across all the files.  There may be one or two places where I missed something.